Update- security

WT-Liu opened the pull request:
<!--
Thank you for contributing to TiKV!

If you haven't already, please read TiKV's [CONTRIBUTING](https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md) document.

If you're unsure about anything, just ask; somebody should be along to answer within a day or two.

PR Title Format:
1. module [, module2, module3]: what's changed
2. *: what's changed
-->

### What problem does this PR solve?

This is related to https://github.com/tikv/tikv/pull/7792



### What is changed and how it works?

update security and TiKV logos


### Related changes


### Check List <!--REMOVE the items that are not applicable-->

Tests <!-- At least one of them must be included. -->

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note <!-- bugfixes or new feature need a release note -->
- No release note

dcalvin commented on SECURITY.md:
```suggestion
···
-----BEGIN PGP PUBLIC KEY BLOCK-----
```

dcalvin commented on SECURITY.md:
```suggestion
-----END PGP PUBLIC KEY BLOCK-----
···
```

BusyJay commented:
So what's the difference between it and #7792?

WT-Liu commented:
> So what's the difference between it and #7792?

I addressed the comments you made in https://github.com/tikv/tikv/pull/7792 and updated the TiKV logo in readme. Sorry that I think I made a mistake and couldn't make a commit to https://github.com/tikv/tikv/pull/7792, so I opened a new one.

dcalvin commented:
/merge
